source/customization.rst

```diff
@@ -1,32 +1,49 @@
 .. _customization:
 
-Customization
-=============
+Customization (or the many OOD environment variables)
+=====================================================
 
-Announcements
----------------
+Setting the PUN Environment With ``nginx_stage.yml``
+----------------------------------------------------
 
-.. attention:: TODO
+The configuration file ``nginx_stage.yml`` is the best way to set the environment which the PUN will use. Defining the mapping ``pun_custom_env`` allows setting Open OnDemand specific variables and is used for branding, message of the day setting, quota warnings, whitelists and other uses. For environment variables that OOD applications may need, are already in the environment and should be exposed to OOD (``$PATH``, ``$LD_LIBRARY_PATH``, etc...) define the sequence ``pun_custom_env_declarations``.
+
+An example of both of these uses may be found in `nginx_stage_example.yml <https://github.com/OSC/ondemand/blob/d85a3982d69746144d12bb808d2419b42ccc97a1/nginx_stage/share/nginx_stage_example.yml#L26-L43>`__
+
+Announcements and Message of the Day (MOTD)
+-------------------------------------------
```

Comment:
I would split these into separate sections as they really are separate features that aren't related to each other at all.